pkcs12.go

```diff
@@ -583,7 +583,33 @@ func EncodeTrustStore(rand io.Reader, certs []*x509.Certificate, password string
 
 	var certBags []safeBag
 	for _, cert := range certs {
-		certBag, err := makeCertBag(cert.Raw, certAttributes)
+
+		bmpFriendlyName, err := bmpString(cert.Subject.CommonName)
```

Comment:
I think we should use `cert.Subject.String()` instead, as the common name is often not helpful (e.g. <https://crt.sh/?caid=183267>'s CN is just "R3"). Thoughts?

Reply:
Agreed - I went for subject initially as I didn't want the `CN=` parts, but to be honest you spend so little time looking at these, and the full subject is clear and unambigious:
![image](https://user-images.githubusercontent.com/69470240/97975804-8f007e80-1dc1-11eb-8018-b33496d39526.png)

I've force-pushed this change in.